pandas/core/ops/__init__.py

```diff
@@ -311,15 +311,11 @@ def dispatch_to_series(left, right, func, str_rep=None, axis=None):
     -------
     DataFrame
     """
-    # Note: we use iloc to access columns for compat with cases
-    #       with non-unique columns.
-    import pandas.core.computation.expressions as expressions
+    # Get the appropriate array-op to apply to each column/block's values.
+    array_op = get_array_op(func, str_rep=str_rep)
```

Comment:
can you remove the get_array_op call on L325

Reply:
Ah, yes, merge-master mistake